app.py

```diff
@@ -10,9 +10,30 @@ def index():
 def input():
 	return render_template("input.html")
 
+#setting up backend to receive urls
```

Comment:
Change the name from `/scrape` to something more like `/save_url`.

Reply:
good evening @Bansikah  i will just use the` /scrape` for the label for the script to scrap the data from the site then